packages/express/src/build.ts

```diff
@@ -41,7 +46,13 @@ export const findEntrypoint = (files: Files) => {
   });
 
   if (!entrypoint) {
-    throw new Error('No valid entrypoint found');
+    throw new Error(`No valid entrypoint found. Valid entrypoints are:
+
+- app.{cjs,js,mjs,mts,ts}
```

Comment:
I realize these are alphabetical, but I'm not sure if that's the easiest way sort them. Up to you if sorting by lang then module is easier:
```suggestion
- app.{js,cjs,mjs,ts,mts}
```